src/browser.mts

```diff
@@ -1,65 +1,53 @@
 import { Fetcher } from './fetcher/fetcher.mjs';
 import { FetchConfig } from './fetcher/types.mjs';
 import { paths } from './types.mjs';
+import * as Sentry from '@sentry/browser';
+import { CaptureConsole } from '@sentry/integrations';
+import { normalizeEnvironmentName } from './utils.mjs';
 
-// This is a hack to forward console.log to the parent window.
-const setupLogForwarding = (apiUrl: string) => {
-  if (!window.parent) {
-    console.warn('No parent window found. Not forwarding logs.');
-    return;
-  }
-  const origin = new URL(apiUrl).origin;
-
-  const proxiedMethods = [
-    'log',
-    'error',
-    'info',
-    'trace',
-    'warn',
-    'debug',
-  ] as const;
-
-  console = new Proxy(console, {
-    get: function (target, prop, receiver) {
-      const method = prop as typeof proxiedMethods[number];
-      if (proxiedMethods.includes(method)) {
-        return function (...rest: any[]) {
-          // window.parent is the parent frame that made this window
-          window.parent.postMessage(
-            {
-              source: 'iframe',
-              message: {
-                type: prop,
-                payload: rest,
-              },
-            },
-            origin
-          );
-          target[method].apply(rest);
-        };
-      }
-      return Reflect.get(target, prop, receiver);
-    },
-  });
-};
-
-const PlaytBrowserClient = ({ apiUrl }: { apiUrl: string }) => {
-  setupLogForwarding(apiUrl);
-
+const PlaytBrowserClient = ({
+  gameId,
+  apiUrl,
+}: {
+  gameId: string;
+  apiUrl: string;
+}) => {
   const fetcher = Fetcher.for<paths>();
   const config: FetchConfig = {
     baseUrl: apiUrl,
     use: [],
+    init: {
+      headers: {
+        'User-Agent': `playt-browser-client/${process.env.npm_package_version}`,
+      },
+    },
   };
   fetcher.configure(config);
 
+  const initialize = async ({ gameVersion }: { gameVersion: string }) => {
+    const sentryConfigResp = await fetcher
+      .path('/api/games/{gameId}/sentry-config')
+      .method('get')
+      .create()({ gameId });
+    if (!sentryConfigResp.ok) {
+      throw new Error('Failed to fetch sentry config');
```

Comment:
should we probably make this not an error? I wonder if this makes life unnecessarily hard for the game devs..